packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -442,7 +442,7 @@ class FlutterErrorDetails extends Diagnosticable {
 
   @override
   String toStringShort() {
-    return library != null ? 'Exception Caught By $library' : 'Exception Caught';
```

Comment:
It will be good to have a test like [this one](https://github.com/flutter/flutter/blob/aa9a115181ce5795f1618f2e032cc771896cea9f/packages/flutter_tools/test/general.shard/commands/test_test.dart#L26) to ensure not one breaks this